tests/com/tony/server/AuthenticatorTest.java

```diff
@@ -0,0 +1,74 @@
+package com.tony.server;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class AuthenticatorTest {
+    @Test
+    public void testAddToAuthenticationList() throws Exception {
```

Comment:
I think I googled something along the lines of best practices for naming junit tests and that's what they told me to do. Do people do something different usually?